examples/extensions/add_business_profile_location_extensions.rb

```diff
@@ -15,57 +15,57 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# This example adds a feed that syncs feed items from a Google My Business (GMB)
+# This example adds a feed that syncs feed items from a Business Profile
 # account and associates the feed with a customer.
 
 require 'optparse'
 require 'google/ads/google_ads'
 require 'date'
 
-def add_google_my_business_location_extensions(
+def add_business_profile_location_extensions(
   customer_id,
-  gmb_email_address,
-  gmb_access_token,
+  business_profile_email_address,
```

Comment:
this should be "business_profile_email" without "_address" suffix. Please see
https://github.com/googleads/google-ads-php/blob/main/examples/Extensions/AddBusinessProfileLocationExtensions.php#L53

Reply:
Please update before merge.